common/app/model/CardStylePicker.scala

```diff
@@ -9,19 +9,23 @@ import commercial.targeting.CampaignAgent
 
 object CardStylePicker {
 
-  def apply(content: CapiContent, trailMetaData: MetaDataCommonFields): CardStyle = {
+  def apply(content: CapiContent): CardStyle = {
     val tags = content.tags.map(_.id).toSeq
     extractCampaigns(tags) match {
       case Nil => CardStyle(content, TrailMetaData.empty)
-      case _   => SpecialReport
+      case campaign :: _ =>
+        if (campaign.id.toString.toLowerCase() == "specialreportalt") SpecialReportAlt else SpecialReport
+      case _ => SpecialReport
     }
   }
 
   def apply(content: FaciaContent): CardStyle = {
     val tags = FaciaContentUtils.tags(content).map(_.id)
     extractCampaigns(tags) match {
       case Nil => FaciaContentUtils.cardStyle(content)
-      case _   => SpecialReport
+      case campaign :: _ =>
+        if (campaign.id.toString.toLowerCase() == "specialreportalt") SpecialReportAlt else SpecialReport
```

Comment:
The campaign information is coming from [`extractCampaigns`](https://github.com/guardian/frontend/blob/main/common/app/model/CardStylePicker.scala#L28) --> [`CampaignAgent`](https://github.com/guardian/frontend/blob/main/common/app/commercial/targeting/CampaignAgent.scala#L39) --> [`targeting-client`](https://github.com/guardian/targeting-client/blob/main/src/main/scala/com/gu/targeting/client/Campaign.scala#L62). There is already a [campaign set up](https://targeting.gutools.co.uk/campaigns/a558a3f1-1963-45f8-90c9-0bd8cc27ea96) with this campaign id in the targeting-tool